src/frontends/pytorch/src/op_table.cpp

```diff
@@ -77,6 +78,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"aten::elu", op::translate_elu},
         {"aten::embedding", op::translate_embedding},
         {"aten::eq", op::translate_1to1_match_2_inputs<opset8::Equal>},
+        {"aten::exp", op::translate_exp},
```

Comment:
Why not to use `op::translate_1to1_match_1_inputs`?

Reply:
updated